clients/java/pom.xml

```diff
@@ -162,6 +162,23 @@
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>clirr-maven-plugin</artifactId>
+        <configuration>
+          <excludes>
+            <exclude>io/zeebe/client/impl/**</exclude>
+          </excludes>
+          <ignored>
+            <ignored>
+              <!-- ignore new methods in the Zeebe element instance interfaces -->
+              <className>io/zeebe/client/util/Environment</className>
```

Comment:
Correct me if I'm wrong, but those classes have been moved under `impl`, so we don't need to ignore anything here anymore, right?

Reply:
No, we still should ignore this, because of Сlirr thinking that we removed the `Environment` class. I was tried to add the `util` package into the excluded tag, but I'm not sure that this is a proper solution :)

Reply:
Can we add this to the `ignored-changes.xml` file instead? This is only affecting the current version, and our release process will remove the ignore-changes.xml at the end.

Reply:
Ok, I'll move this in the `ignored-changes.xml`

Reply:
Done